service/controller/resource/resource.go

```diff
@@ -150,6 +154,47 @@ func New(config Config) ([]resource.Interface, error) {
 		}
 	}
 
+	var remoteWriteAgentSecretResource resource.Interface
+	{
+		c := remotewriteagentsecret.Config{
+			K8sClient:       config.K8sClient,
+			Logger:          config.Logger,
+			PasswordManager: password.SimpleManager{},
+		}
+
+		remoteWriteAgentSecretResource, err = remotewriteagentsecret.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var remoteWriteAgentConfigSecretResource resource.Interface
+	{
+		c := remotewriteagentconfigsecret.Config{
+			K8sClient:  config.K8sClient,
+			Logger:     config.Logger,
+			BaseDomain: config.PrometheusBaseDomain,
+		}
+
+		remoteWriteAgentConfigSecretResource, err = remotewriteagentconfigsecret.New(c)
+		if err != nil {
+			return nil, microerror.Mask(err)
+		}
+	}
+
+	var remoteWriteSecretResource resource.Interface
```

Comment:
I wonder, the `remotewritesecret` package for creating secrets from RemoteWrite CR. `spec.secrets`
Why this is needed?

Reply:
I think it's not and it's a wrong import